design/authn_azure/authn_azure_solution_design.md

```diff
@@ -420,44 +420,44 @@ Azure authenticator performance should conform with our other authenticators wit
-| 16 | One or more fields in the Azure token don't match the Conjur host annotations                                               | VM sends authentication request to Conjur with valid Azure token    | Return 401 Unauthorized error and `InvalidApplicationIdentity` message appears in log    | Integration               | [ ]            |
+|    | Given                                                                                                                       | When                                                                | Then                                                                                     | Manual / UT / Integration | Status                       |Core Test  |
+|----|-----------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------|------------------------------------------------------------------------------------------|---------------------------|------------------------------|------------
+| 1  | *Vanilla flow* - a host identity exists for Azure VM and added to privileged group for authentication on webservice         | VM sends authentication request with a valid Azure token            | Conjur access token is returned in body and action is logged                             | Integration               | <ul><li> [ ] Tested</li></ul>| Yes       |
+| 2  | *Vanilla flow* - a user identity exists for Azure VM and added to privileged group for authentication on webservice         | VM sends authentication request with a valid Azure token            | Conjur access token is returned in body and action is logged                             | Integration               | <ul><li> [ ] Tested</li></ul>| Yes       |
+| 3  | A host identity for Azure VM has been successfully authorized by Conjur                                                     | VM sends request with valid Azure token to fetch secret             | Azure resource can successfully send GET request for a secret and action is logged       | Integration               | <ul><li> [ ] Tested</li></ul>| Yes       |
+| 4  | A host identity for Azure VM has not been added to privileged group                                                         | VM sends request with valid Azure token                             | Return 401 Unauthorized error and `RoleNotAuthorizedOnWebservice` and action is logged   | Integration               | <ul><li> [ ] Tested</li></ul>| Yes       |
+| 5  | A host identity for Azure VM has been added to group that only has read OR authenticate permissions on webservice           | VM sends request with valid Azure token                             | Return 401 Unauthorized error and `RoleNotAuthorizedOnWebservice` and action is logged   | Integration               | <ul><li> [ ] Tested</li></ul>| No        |
```

Comment:
This test is incorrect. If the host has `authenticate` permissions on the webservice it will get an access token. `read` permissions are not mandatory for authentication, it just enhances UX as when you list resources with the host you will see that it can authenticate.

Second, I think that this should not be a core test as it's not an Azure authenticator flow. It's part of the general flow.

Reply:
Done